src/actions/channels.js

```diff
@@ -209,10 +217,6 @@ export function createGroupChannel(userIds) {
                 type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBER,
                 data: member,
             },
-            {
```

Comment:
I'm guessing `savePreferences` already effected this action?

Reply:
That action is being received by the websocket if i’m not mistaken

Reply:
Yes, it already effects this through the redux offline dispatch